Services/Simulations.cs

```diff
@@ -124,6 +165,7 @@ public class Simulations : ISimulations
         }
 
         /// <summary>
+        /// Create or Replace a simulation
```

Comment:
Nit. Summary content for Upsert below has punctuation. "Upsert the simulation" may also be redundant now with the added summary content.

Reply:
👍